src/main/java/org/bytedeco/javacpp/Pointer.java

```diff
@@ -589,6 +590,10 @@ public static long maxPhysicalBytes() {
     /** Returns the amount of physical memory that is free according to the operating system, or 0 if unknown. */
     @Name("JavaCPP_availablePhysicalBytes") public static native long availablePhysicalBytes();
 
+    /** Returns the starting address of the memory region referenced by the given direct {@link java.nio.Buffer}.
+     * An alternative to allocating a new Pointer object if all you need is the memory address. */
+    @Name("JavaCPP_getDirectBufferAddress") public static native long getDirectBufferAddress(@Raw(withEnv = true) Buffer b);
```

Comment:
Can we make this protected? It doesn't feel like something that should be used outside of a wrapper.

Reply:
By wrapper, do mean a class that extends Pointer? I suppose that's a good point, since we're just using this to set the address of our MutableBytePointer class, which allows publicly setting the address. We can just expose a method in that one that uses this internally.